vmdb/app/models/miq_server/rhn_mirror.rb

```diff
@@ -1,4 +1,5 @@
 require 'linux_admin'
+require 'miq_apache'
```

Comment:
@kbrock 
These ruby files use the MiqApache functions, previously it was autoloaded by rails, now we would have to make it explicit